services/export/config.js

```diff
@@ -57,4 +63,15 @@ config.csv.delimiter = ';';
 config.csv.dateFormat = 'YYYY-MM-DD';
 config.csv.timeFormat = 'hh:mm:ss';
 
+config.queue = {
+	defaults: {
+		defaultJobOptions: {
```

Comment:
Please consider renaming the property.
```suggestion
		jobOptions: {
```